resources-sample.tsv

```diff
@@ -1,2 +1,3 @@
-max_file_size	cores mem 	time
-"0"	"8"	"16000"	"00-18:00:00"
\ No newline at end of file
+max_file_size	cores	mem	time
+0.001	1	8000	00-18:00:00
+NaN	1	16000	01-00:00:00
```

Comment:
does this really require 16G of memory? this seems like a lot...

Reply:
> does this really require 16G of memory? this seems like a lot...

Some of the encode files needed more than 16G.  

I changed the code from using pandas to awk.  I still need to test how much memory it needs after the change.